config/package-bundle/config/crds.yml

```diff
@@ -40,6 +40,30 @@ spec:
                 items:
                   type: string
                 type: array
+              toSelectorMatchFields:
```

Comment:
This field name does not provide much information about what is being selected here. I think we should rename this.
My main concern about this field is that we provide a foot gun to the users. If someone defines a selector for an annotation to allow a copy of secrets, a third-party actor creating or editing namespaces can funnel all the secrets to that namespace, which can be dangerous.

Since this opens the users to possible problems, I think we can change this property name to `dangerousToNamespacesSelector`

Also, looking at the way these some Kubernetes resources use this type of selector, maybe it would make more sense for us to have something similar to:

```yaml
dangerousToNamespacesSelector:
  matchExpressions:
    - { key: tier, operator: In, values: [cache] }
    - { key: environment, operator: NotIn, values: [dev] }
    - { key: environment, operator: Exists }
```

What do yll think?